projects/go-lib/src/lib/components/go-loader/go-loader.component.ts

```diff
@@ -1,40 +1,33 @@
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, HostBinding, Input } from '@angular/core';
+import { fadeTemplateAnimation } from '../../animations/fade.animation';
 
 @Component({
   selector: 'go-loader',
   templateUrl: './go-loader.component.html',
-  styleUrls: ['./go-loader.component.scss']
+  styleUrls: ['./go-loader.component.scss'],
+  animations: [
+    fadeTemplateAnimation
+  ]
 })
-export class GoLoaderComponent implements OnInit {
-
-  @Input() loaderDone: boolean = false;
+export class GoLoaderComponent {
   @Input() loaderSize: string = 'medium';
   @Input() loaderType: string = 'neutral';
 
-  constructor() { }
-
-  ngOnInit() {
-  }
+  @HostBinding('@fadeTemplate')
+  public fadeTemplate: boolean = true;
 
-  completeAnimation() : void {
-    this.loaderDone = true;
-  }
+  //#region Public Methods
 
-  public pathClasses() : object {
+  containerClasses(): object {
```

Comment:
We probably don't need to call this `containerClasses` anymore because the `go-loader__container` class was removed. Now these are just `loaderClasses`. Not a necessary change, of course. I'll leave it up to you.